docs/reference/components/catalog.md

```diff
@@ -0,0 +1,385 @@
+
+
+## Writing a Catalog Spec
+
+### Sources
+
+`.spec.sources` is a list of sources from which the Catalog will fetch PluginDefinition manifests. Currently, only Git repositories are supported as sources.
+Each source requires a `repository` URL and a list of `resources` that specify the paths to the PluginDefinition manifests within the repository. 
+The `ref` field is used to specify the branch, tag, or commit SHA to fetch from the repository.
+
+⚠️  Each Organization has a dedicated ServiceAccount used to apply the Catalog resources. This ServiceAccount only has permissions to apply PluginDefinitions into the Organization's Namespace. It will not be possible to bring other Kinds using the Catalog.
+
+### Repository
+
+`.spec.sources[].repository` is used to specify the URL of the Git repository containing the PluginDefinition manifests.
+
+### Resources
+
+`.spec.sources[].resources` is a list of file paths within the repository that point to the PluginDefinition manifests to be included in the Catalog.
+
+> Empty resources list will result in an error.
+
+### Ref
+
+`.spec.sources[].ref` is used to specify the branch, tag, or commit SHA to fetch from the repository.
+
+Available fields are:
+
+- **sha** - The commit SHA to fetch.
+- **tag** - The tag to fetch.
+- **branch** - The branch that is targeted.
+
+The priority order is: _sha_ > _tag_ > _branch_. If multiple fields are specified, the field with the highest priority will be used.
+
+> When multiple sources are defined with the same repository and ref, a duplicate error will be raised.
+
+### Interval (Optional)
+
+`.spec.sources[].interval` is an optional field that specifies how often the Catalog should check for updates in the source repository.
+The default value is `1h` (1 hour) if not specified.
+
+The value must be in a [Go recognized duration format](https://pkg.go.dev/time#ParseDuration), e.g. `5m0s` to reconcile the source every 5 minutes.
+
+### Secret Reference (Optional)
+
+`.spec.sources[].secretName` is an optional field that specifies a reference to a Kubernetes Secret name containing credentials for accessing private Git repositories.
+
+The following are the types of authentication supported:
+
+- [Basic Authentication](#configuring-secret-for-basic-authentication)
+- [GitHub App Credentials](#configure-secret-for-github-app-authentication)
+
+The Secret must be in the same namespace as the Catalog resource.
+
+### Overrides (Optional)
+
+`.spec.sources[].overrides` is an optional field that allows specifying overrides for specific PluginDefinitions in the Catalog. 
+This can be used to customize certain fields of PluginDefinitions.
+
+Currently, you can override the following fields:
+
+- `alias` field to override `metadata.name` of the PluginDefinition
+- `repository` field to override the helm chart repository in the PluginDefinition spec
+
+## Working with Catalog
```

Comment:
Could you add similar sections for suspend/reconcile annotations for the catalog in the working with catalog section?https://cloudoperators.github.io/greenhouse/docs/reference/components/plugin/#suspending-the-plugins-reconciliation